code/datums/autolathe/guns.dm

```diff
@@ -115,11 +115,11 @@
 	build_path = /obj/item/gun/projectile/lamia/dark
 
 /datum/design/autolathe/gun/scoped_naga
-	name = "HG .50 Kurtz \"naga\" Advanced"
+	name = "SolFed HG .50 Kurtz \"Apollo\" Advanced"
```

Comment:
'Apollo' if I recall is the name of one of Kavrick's weapons. Maybe just call it 'Gemini,' another space mission?